### PR TITLE
Remove subscribe to TLS.

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge PR

### CODE REVIEW

File: searchcore/src/vespa/searchcore/proton/server/feedhandler.h [hunk 1]
```diff
@@ -184,22 +184,19 @@ class FeedHandler: private search::transactionlog::TransLogClient::Session::Call
                 search::transactionlog::Writer & writer,
                 TlsWriter * tlsWriter = nullptr);
 
-    virtual
-    ~FeedHandler();
+    virtual~FeedHandler();
```
Comment (toregge) [inline searchcore/src/vespa/searchcore/proton/server/feedhandler.h]: missing space

File: searchcore/src/vespa/searchcore/proton/server/feedhandler.h [hunk 2]
```diff
@@ -270,46 +259,18 @@ class FeedHandler: private search::transactionlog::TransLogClient::Session::Call
...
-    void
-    storeRemoteOperation(const FeedOperation &op);
+    RPC::Result receive(const Packet &packet) override;
 
-    // Implements IOperationStorer
-    virtual void storeOperation(FeedOperation &op) override;
+    void eof() override;
+    void performPruneRemovedDocuments(PruneRemovedDocumentsOperation &pruneOp) override;
+    void syncTls(SerialNum syncTo);
+    void storeRemoteOperation(const FeedOperation &op);
```
Comment (toregge) [inline searchcore/src/vespa/searchcore/proton/server/feedhandler.h]: This method can probably be removed.

**Comment by baldersheim** [comment]
Comments addressed, I also removed some other unused code in FeedHandler.

**Comment by baldersheim** [comment]
Now I think it is ready to go in.

**Comment by toregge** [comment]
LGTM

**Pull request merged**